src/test/java/com/mariamkatamashvili/trainerwork/messaging/MessageConsumerTest.java

```diff
@@ -0,0 +1,45 @@
+package com.mariamkatamashvili.trainerwork.messaging;
```

Comment:
Would be much better to introduce package 'cucumber' and store all relative stuff inside